ibis/backends/base/sql/alchemy/translator.py

```diff
@@ -40,12 +43,40 @@ class AlchemyExprTranslator(ExprTranslator):
 
     context_class = AlchemyContext
 
+    _bool_aggs_need_cast_to_int32 = True
+    _has_filter_syntax = False
```

Comment:
Can you add `reduction` to this config option otherwise it's easy to misinterpret.

Reply:
Done.